### ISSUE TITLE
Anormal Profit Results

### ISSUE THREAD

**Issue by mablue** [freqtrade/freqtrade#4637]
## Describe your environment

  * Operating system: Ubuntu 20
  * Python Version: 3.8.5
  * CCXT version: 1.45.44 
  * Freqtrade Version: develop-00d8f3a4

## Describe the problem:

The result will be increase but  1 and 2 is very anormal and it happend in other strategys to.(strategys that have not good win/loss some times have very huge profit!)

### Steps to reproduce:

  1. editing ADXMomentum strategy:

>     # ROI table:
>     minimal_roi = {
>         "0": 0.47327,
>         "388": 0.12599,
>         "954": 0.05119,
>         "1777": 0
>     }
> 
>     # Stoploss:
>     stoploss = -0.34478
> 
>     # Trailing stop:
>     trailing_stop = True
>     trailing_stop_positive = 0.34626
>     trailing_stop_positive_offset = 0.37694
>     trailing_only_offset_is_reached = True

2. hyperopt by empty ho
3. edit the config like that:
```
{
    "max_open_trades": 30,
    "stake_currency": "USDT",
    "stake_amount": 100,
    "tradable_balance_ratio": 0.9,
    "fiat_display_currency": "USD",
    "timeframe": "1h",
    "dry_run": true,
    "dry_run_wallet": 3700,
    "cancel_open_orders_on_exit": false,
    "unfilledtimeout": {
        "buy": 10,
        "sell": 30
    },
    "bid_strategy": {
        "price_side": "bid",
        "ask_last_balance": 0.0,
        "use_order_book": false,
        "order_book_top": 1,
        "check_depth_of_market": {
            "enabled": false,
            "bids_to_ask_delta": 1
        }
    },
    "ask_strategy": {
        "price_side": "ask",
        "use_order_book": false,
        "order_book_min": 1,
        "order_book_max": 1,
        "use_sell_signal": true,
        "sell_profit_only": false,
        "ignore_roi_if_buy_signal": false
    },
    "exchange": {
        "name": "binance",
        "key": "",
        "secret": "",
        "ccxt_config": {
            "enableRateLimit": true
        },
        "ccxt_async_config": {
            "enableRateLimit": true
        },
        "pair_whitelist": [
            ".*/USDT",
        ],
        "pair_blacklist": [
            "EOSC/USDT",
        ],
        "pair_blacklist": []
    },
    "pairlists": [
        {
            "method": "StaticPairList"
        }
    ],
    "edge": {
        "enabled": false,
        "process_throttle_secs": 3600,
        "calculate_since_number_of_days": 7,
        "allowed_risk": 0.01,
        "stoploss_range_min": -0.01,
        "stoploss_range_max": -0.1,
        "stoploss_range_step": -0.01,
        "minimum_winrate": 0.60,
        "minimum_expectancy": 0.20,
        "min_trade_number": 10,
        "max_trade_duration_minute": 1440,
        "remove_pumps": false
    },
    "telegram": {
        "enabled": true,
        "token": "",
        "chat_id": ""
    },
    "api_server": {
        "enabled": false,
        "listen_ip_address": "127.0.0.1",
        "listen_port": 8080,
        "verbosity": "error",
        "enable_openapi": false,
        "jwt_secret_key": "somethingrandom",
        "CORS_origins": [],
        "username": "",
        "password": ""
    },
    "bot_name": "freqtrade",
    "initial_state": "running",
    "forcebuy_enable": false,
    "internals": {
        "process_throttle_secs": 5
    }
}
```
4. see the result lines 1 and 2:
ADXMomentum
```
+--------+---------+----------+------------------+--------------+-------------------------------------+----------------+-------------+
|   Best |   Epoch |   Trades |    Win Draw Loss |   Avg profit |                              Profit |   Avg duration |   Objective |
|--------+---------+----------+------------------+--------------+-------------------------------------+----------------+-------------|
| * Best |   1/500 |     1260 |    536  251  473 |      305.04% | 384,736.45275716 USDT (384,352.10%) |        917.3 m |     -3.4965 |
| * Best |   2/500 |    11124 |   4175   20 6929 |       34.71% | 386,526.62644591 USDT (386,140.49%) |         45.1 m |    -3.50702 |
| * Best |   3/500 |      838 |    339  384  115 |        1.24% | 1,037.47609053 USDT (1,036.44%) |      1,421.7 m |     -7.9087 |
| * Best |   4/500 |      703 |    363  221  119 |        1.65% | 1,159.50474967 USDT (1,158.35%) |      1,705.5 m |    -8.03313 |
| * Best |   5/500 |      656 |    394  126  136 |        1.71% | 1,125.13846992 USDT (1,124.01%) |      1,832.5 m |    -9.01747 |
| * Best |   7/500 |      958 |    571  235  152 |        1.49% | 1,425.20603870 USDT (1,423.78%) |      1,236.1 m |    -9.68559 |
| * Best |  21/500 |      765 |    369  276  120 |        1.97% | 1,505.01737964 USDT (1,503.51%) |      1,563.2 m |    -10.0827 |
|   Best |  76/500 |      864 |    443  257  164 |        1.66% | 1,432.74247238 USDT (1,431.31%) |      1,376.4 m |    -10.3254 |
|   Best |  79/500 |      779 |    427  196  156 |        1.86% | 1,450.21223563 USDT (1,448.76%) |      1,533.8 m |    -11.4308 |
|   Best | 178/500 |      758 |    408  210  140 |        2.22% | 1,683.78285746 USDT (1,682.10%) |      1,577.9 m |    -12.7322 |
|   Best | 286/500 |      893 |    482  259  152 |        2.44% | 2,183.40057714 USDT (2,181.22%) |      1,330.1 m |    -13.0972 |
|   Best | 298/500 |      920 |    547  218  155 |        2.30% | 2,120.11207496 USDT (2,117.99%) |      1,288.8 m |    -14.6112 |
|   Best | 305/500 |      836 |    453  238  145 |        2.60% | 2,174.42621196 USDT (2,172.25%) |      1,424.6 m |    -14.9097 |
|   Best | 309/500 |      875 |    509  234  132 |        2.80% | 2,450.22506989 USDT (2,447.78%) |      1,358.4 m |    -16.6732 |
|   Best | 312/500 |     1184 |    785  239  160 |        2.32% | 2,749.12153124 USDT (2,746.38%) |        988.1 m |    -16.8309 |
|   Best | 313/500 |     1265 |    910  200  155 |        1.85% | 2,347.70910213 USDT (2,345.36%) |        920.7 m |    -17.3198 |
|   Best | 315/500 |     1625 |   1302  175  148 |        1.96% | 3,189.85438530 USDT (3,186.67%) |        703.4 m |    -19.6109 |
|   Best | 319/500 |     1578 |   1194  241  143 |        2.24% | 3,539.01965180 USDT (3,535.48%) |        726.5 m |    -20.0861 |
|   Best | 320/500 |     1850 |   1536  159  155 |        1.97% | 3,648.94063946 USDT (3,645.30%) |        610.5 m |    -21.8669 |

```

**Comment by xmatthias** [comment]
most likely, it'll result in parameters that exploit one of the [backtesting assumptions](https://www.freqtrade.io/en/latest/backtesting/#assumptions-made-by-backtesting) (most likely very tight trailing stoploss ...)

Please have a look at the resuting parameters yourself and try to investigate yourself. 

What you can do:
convert the result to strategy (probably both?) and run a regular backtest on it.
the sell-reasons will give you a clue, usually (if almost everything is from trailing stop, that's often the reason).


Keep in mind that this is a free, open source project. It's people's free time that goes into answering issues, so please make sure you did good research before posting an issue.
